docs/UserGuide.md

```diff
@@ -126,7 +126,7 @@ Format: `clear`
 
 ### Exiting the Program: `exit`
 
-Automatically saves all of the user's client data and exits the program.
```

Comment:
By any chance, was the original version of this, with the 'of' better?

Reply:
ok will change it back